### ISSUE TITLE
Admin\User and Access Requests - External User can view operator access requests

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#1716]
#### Description:
As an external user, I want to see the access requests for my operator

#### Acceptance Criteria:

Given I am on the [user and access requests grid](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=18084-6815&t=irsws3GIrtxrdGhp-4) #1714 
I can see access requests 

Given that I have approved of an access request
Then I can see a success snackbar that says "user is now approved"
and the approved user can now access my operator

Given that I declined an access request
Then I can see a success snackbar that says user has been declined
and the user cannot access my operator

Given that I had clicked either approved or declined on an access request
Then an "Edit" button replaces the approve/decline buttons
And if I click "Edit" then the approve/decline buttons come back

#### Development Checklist:

- [x] Add a `SnackBar` with the appropriate message to the `ActionColumnCell.tsx` component in `bciers/apps/administration/app/components/userOperators/cells/` after handling `handleAccessRequestStatus` in the `onClick` prop.
- [x] update vitest tests in `bciers/apps/administration/tests/components/userOperators/AccessRequestDataGrid.test.tsx`
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by
- Blocking

**Comment by Sepehr-Sobhani** [comment]
Is this a duplicate of the ticket https://github.com/bcgov/cas-registration/issues/1714 ?

**Comment by zoeyli-46** [comment]
Hey @Sepehr-Sobhani in #1714, it doesnt have the approve and decline access requests messages. unless, are these already implemented from reg 1?
"Given that I have approved of an access request
Then I can see a success snackbar that says "user is now approved"
and the approved user can now access my operator

Given that I declined an access request
Then I can see a success snackbar that says user has been declined
and the user cannot access my operator"

**Comment by patrickisaac** [comment]
Refinement Notes:
- Dev Checklist still required (@Sepehr-Sobhani to provide)
- No email stuff

**Comment by Sepehr-Sobhani** [comment]
We have already implemented sending emails to users to inform them about the request status using the 'send_operator_access_request_email' function.
cc: @patriciarussellCAS / @andrea-williams

**Comment by andrea-williams** [comment]
@zoeyli-46 the wireframe linked in this ticket takes me to the COAM board in Figma. Could you update the link please?

**Comment by zoeyli-46** [comment]
its updated

**Comment by patriciarussellCAS** [comment]
This worked as intended. thanks!
